clippy_lints/src/needless_bool.rs

```diff
@@ -413,7 +353,7 @@ fn suggest_bool_comparison<'a, 'tcx>(
         BOOL_COMPARISON,
         span,
         message,
-        "try simplifying it as shown",
+        "try simplifying it",
```

Comment:
Usually, more direct instructions such as "simplify it" or "use" are better than "try doing …" (which is the historical way of giving instructions).

(two places)